OpenEdXMobile/res/layout/activity_register.xml

```diff
@@ -3,120 +3,135 @@
+                    android:layout_height="wrap_content"
+                    android:orientation="vertical"
+                    android:visibility="gone" />
 
-            <LinearLayout
-                android:id="@+id/optional_fields_layout"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:orientation="vertical"
-                android:visibility="gone" />
+                <FrameLayout
+                    android:id="@+id/createAccount_button_layout"
+                    style="@style/edX.Widget.SignInButtonLayout"
+                    android:contentDescription="@string/create_account_btn"
+                    android:focusable="true">
+
+                    <TextView
+                        android:id="@+id/create_account_tv"
+                        style="@style/edX.Widget.SignInButton"
+                        android:text="@string/create_account_text" />
 
-            <FrameLayout
-                style="@style/edX.Widget.SignInButtonLayout"
-                android:focusable="true"
-                android:id="@+id/createAccount_button_layout"
-                android:contentDescription="@string/create_account_btn">
+                    <include layout="@layout/button_progress_indicator" />
+                </FrameLayout>
 
                 <TextView
-                    style="@style/edX.Widget.SignInButton"
-                    android:id="@+id/create_account_tv"
-                    android:text="@string/create_account_text" />
+                    android:id="@+id/by_creating_account_tv"
+                    style="@style/regular_text"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="8dp"
+                    android:gravity="center"
+                    android:textColor="@color/grey_redirected_txt"
+                    android:textSize="11sp"
+                    tools:text="@string/by_creating_account" />
+
+                <LinearLayout
+                    android:id="@+id/layout_agreement"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:orientation="vertical" />
 
-                <include layout="@layout/button_progress_indicator" />
-            </FrameLayout>
+            </LinearLayout>
+        </ScrollView>
 
-            <TextView
-                android:id="@+id/by_creating_account_tv"
-                style="@style/regular_text"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:layout_marginTop="8dp"
-                android:gravity="center"
-                tools:text="@string/by_creating_account"
-                android:textColor="@color/grey_redirected_txt"
-                android:textSize="11sp" />
+        <TextView
```

Comment:
Its coupled with the usage in `BaseFragmentActivity` class which `RegisterActivity` extends

**Usage:** https://github.com/edx/edx-app-android/blob/farhan/LEARNER-299/OpenEdXMobile/src/main/java/org/edx/mobile/base/BaseFragmentActivity.java#L362